.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,23 +17,42 @@ Steps to reproduce the behavior:
 3. Scroll down to '....'
 4. See error
 
-**Which AWS service(s) are affected?**
+**Observed Behavior**
 
-**Expected behavior**
+**Expected Behavior**
 A clear and concise description of what you expected to happen.
 
+**Stack Trace**
+Please provide a stack trace if applicable i.e. a crash is occurring.
+
+**Code Snippet**
+Please provide a snippet of the code causing the issue or how you are using the service that has the bug.
+
+**Unique Config**
+If you are reporting an issue with a unique configuration or where configuration can make a difference in code execution (i.e. Cognito) please provide your configuration.
```

Comment:
Mention them to remove sensitive information from the configuration is required?

Reply:
fixed